src/binary/ibc-relayer/commands/start.ts

```diff
@@ -17,6 +17,17 @@ import { resolveKeyFileOption } from '../../utils/options/shared/resolve-key-fil
 import { resolveMnemonicOption } from '../../utils/options/shared/resolve-mnemonic-option';
 import { signingClient } from '../../utils/signing-client';
 
+type LoopOptions = {
+  // how many seconds we sleep between relaying batches
+  poll: number;
+  // number of seconds old the client on chain A can be
+  maxAgeSrc: number;
+  // number of seconds old the client on chain B can be
+  maxAgeDest: number;
+  // if set to 'true' quit after one pass
+  once?: boolean;
```

Comment:
The question mark `?` is redundant. Parser always resolves non-parametrized flags to booleans.